src/main/java/codeu/controller/FileUploadServlet.java

```diff
@@ -77,15 +93,27 @@ else if(uri.equals("/profileView/FileUploadServlet")) {
 	public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
-	    /* Title given by user in profile.jsp. */
-        String mediaTitle = (String)request.getParameter("Title"); 
+      	String uri = request.getRequestURI();
+	    
+	    String mediaTitle = null;
+	    if(!uri.equals("/chat/FileUploadServlet")) {
```

Comment:
As a good practice,
Prefer checking truth in conditional rather than negation, meaning

Suppose
foo = !bar
instead of doing
if (!bar) {
}
prefer to do
if (food) {
}